pgcli/completion_refresher.py

```diff
@@ -140,3 +136,7 @@ def refresh_casing(completer, executor):
     if os.path.isfile(casing_file):
         with open(casing_file, 'r') as f:
             completer.extend_casing([line.strip() for line in f])
+
+@refresher('functions')
```

Comment:
@koljonen looks like you just moved the function definition. Can you explain me this ?